cmd/srcd/cmd/components.go

```diff
@@ -47,21 +46,21 @@ var componentsListCmd = &cobra.Command{
 			return humanizef(err, "could not list images")
 		}
 
-		w := tabwriter.NewWriter(os.Stdout, 0, 0, 4, ' ', 0)
-		fmt.Fprintf(w, "IMAGE\tINSTALLED\tRUNNING\tPORT\tCONTAINER NAME\n")
+		header := []string{"IMAGE", "INSTALLED", "RUNNING", "PORT", "CONTAINER NAME"}
+		formats := []string{"%s", "%s", "%v", "%v", "%v"}
+		data := make([][]interface{}, len(cmps))
```

Comment:
just an idea of a little bit different API to avoid types convertions:
```go
table := table.New("%s", "%s", "%v", "%v", "%v")
table.Headers("IMAGE", "INSTALLED", "RUNNING", "PORT", "CONTAINER NAME")
for _, cmp := range cmps {
	table.Row(
		cmp.ImageWithVersion(),
		boolFmt(cmp.IsInstalled()),
		boolFmt(cmp.IsRunning()),
		publicPortsFmt(cmp.GetPorts()),
		cmp.Name,
	)
}

return table.Print(os.Stdout)
```

Reply:
I like it! It's much cleaner than dealing with that `[][]interface{}`.